SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -683,6 +679,46 @@ void UGlobalStateManager::IncrementSessionID()
 	SendSessionIdUpdate();
 }
 
+void UGlobalStateManager::Advance()
+{
+	const TArray<Worker_Op>& Ops = NetDriver->Connection->GetCoordinator().GetViewDelta().GetWorkerMessages();
+
+	ClaimHandler->ProcessOps(Ops);
+	QueryHandler.ProcessOps(Ops);
+
+#if WITH_EDITOR
+	for (const Worker_Op& Op : Ops)
+	{
+		if (Op.op_type != WORKER_OP_TYPE_COMMAND_REQUEST)
+		{
+			continue;
+		}
+
+		const Worker_CommandRequestOp& CommandRequest = Op.op.command_request;
```

Comment:
This command reponse logic seems pretty generic and reusable - did you think about wrapping it in some type of handler that filter against componentid/commandids?

Reply:
Let's discuss this on call as well.

Reply:
Agreed on something like this:
```
template<Worker_ComponentId ComponentId, Worker_CommandIndex CommandIndex>
class CommandRequestHandler
{

	void ProcessOps()
	{
	}
	void Handle(const Worker_CommandRequest& Request)
	{
		if (Request.request.CommandIndex == CommandIndex && Request.request.component_id == ComponentId)
		{
			DispatchHandle(Request);
		}
	}
};
```